godot-core/src/obj/traits.rs

```diff
@@ -210,6 +210,22 @@ pub trait IndexEnum: EngineEnum {
 /// Trait that's implemented for user-defined classes that provide a `Base<T>` field.
 ///
 /// Gives direct access to the containing `Gd<Self>` from `Self`.
+///
+/// # Using WithBaseField as a bound
+///
+/// In order to call `self.base()` or `self.base_mut()` within a trait or on a type you define, the type of `Self::Base` must be specified via `GodotClass<Base = T>`
+///
+/// E.g.
+///
+/// ```no_run
+/// fn some_fn<T>(gd: Gd<T>)
```

Comment:
```suggestion
/// fn some_fn<T>(value: &T)
```
I think value is supposed to be `&T`, instead of `Gd<T>`? `base` is a member of `WithBaseField`, which is derived by `T`, not `Gd`.

Reply:
oh you're right, i made a typo in #715